content/src/content/jcr_root/apps/core/wcm/components/experiencefragment/v1/experiencefragment/_cq_editConfig.xml

```diff
@@ -33,7 +33,7 @@
         <editInNewTab
             jcr:primaryType="nt:unstructured"
             text="Edit"
-            handler="Granite.author.experienceFragments &amp;&amp; Granite.author.experienceFragments.actions.editInNewTab"
+            handler="CQ.CoreComponents.experienceFragment.v1.actions.editInNewTab"
```

Comment:
Can't we simply call the action `CQ.CoreComponents.experiencefragment.v1.actions.edit`, the new tab part seems redundant.

Reply:
Fixed in follow-up commit